runtime/doc/term.txt

```diff
@@ -592,6 +592,17 @@ before using the mouse:
 	"g<LeftMouse>"	is "<C-LeftMouse>	(jump to tag under mouse click)
 	"g<RightMouse>" is "<C-RightMouse>	("CTRL-T")
 
+							*bracketed-paste-mode*
+Bracketed paste mode allows users to paste lines into the terminal emulator
+without Neovim giving any special meaning to it. Most notably it won't try
```

Comment:
I think "into the terminal emulator" is ambiguous here: it means the TUI, but it can be understood as `:terminal`. Perhaps:

> Bracketed paste mode allows users to paste lines into Neovim without it being processed first, when running in the terminal.